Python/compile.c

```diff
@@ -8628,11 +8639,8 @@ assemble(struct compiler *c, int addNone)
     }
     nlocalsplus -= numdropped;
 
-    consts = consts_dict_keys_inorder(c->u->u_consts);
-    if (consts == NULL) {
-        goto error;
-    }
-    if (calculate_jump_targets(g->g_entryblock)) {
+    /** Desugaring **/
```

Comment:
"Convert pseudo instructions to real instructions" maybe?

Reply:
I don't think that's quite right - this is more about translation of jump/exception targets.  There are pseudo instructions that we resolve in the assembly stage (like jump directions).

Shall we call this a preprocessing stage?